docs/guide/normalization.rst

```diff
@@ -84,7 +84,7 @@ Testing
 
 You can use ``e3nn.util.test.assert_normalized`` to check whether a function or module is normalized at initialization:
 
-.. code::
+.. jupyter-execute::
```

Comment:
@mariogeiger I didn't do this since the cell has no output, and since I didn't want to run expensive (relatively) computations on readthedocs

Reply:
(since converging the stats for `Linear` takes larger settings than the default — it fails right now on the docs)

Reply:
Ah ok, let me remove my commit then